Makefile

```diff
@@ -110,6 +110,7 @@ check: test
 cctest: all
 	@out/$(BUILDTYPE)/$@
 
+v8: export GYPFLAGS += -Dclang=0
```

Comment:
This is something I want explicit sign off on... is it ok to use export here?

Reply:
Yep, seems fine.